controllers/organizationsController.js

```diff
@@ -0,0 +1,22 @@
+const { getById } = require('../services/organizationServices')
+const { ErrorObject } = require('../helpers/error')
+const { endpointResponse } = require('../helpers/success')
+const { catchAsync } = require('../helpers/index')
+
+module.exports = {
+  getById: catchAsync(async (req, res, next) => {
+    try {
+      const organization = await getById(req.params.id)
+      /* if (organization) {
+        return next(new ErrorObject('No organization were found with that ID', 404))
+      } */
+      endpointResponse({
+        res,
+        message: 'Organization retrieved successfully.',
+        body: organization,
```

Comment:
Aca te esta pidiendo que le retornes un JSON con unos datos especificos :name, image, phone, address and welcomeText fields.
en el body le vas a tener que pasar ese json con un include para que te traiga la info que especifica

Reply:
`name, image, phone, address y welcomeText`